### PR TITLE
5922-ExtractRecycleBin.parseIFile-makes-assumptions-about-size-of-data-read

### PR DESCRIPTION
**Pull request by markmckinnon**
Add $RECYCLE.BIN as the directory to look for $I files.

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java [hunk 1]
```diff
@@ -266,37 +269,34 @@ private void popuplateDeletedDirectory(SleuthkitCase skCase, AbstractFile parent
...
-     * <tr><td>28</td><td>var</td><td>File Name</td></tr>
-     * </table>
-     *
+     * Offset  Size  Description
+     *   0       8     Header
+     *   8       8     File Size
+     *  16       8     Deleted TimeStamp
+     *  24       4     File Name Length
+     *  28     var     File Name
+     *
```
Comment (rcordovano) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: Why remove the formatting? Did this not build /format properly in doxygen? Does it build /format properly now?

Comment (markmckinnon) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: I will check it out.

Comment (rcordovano) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: Install Doxygen.
http://www.doxygen.nl/download.html
Download and run the Windows installer. 

Run doxygen for Sleuthkit in ~sleuthkit/bindings/java/doxygen, then run doxygen for Autopsy in ~autopsy/docs/doxygen, ~autopsy/docs/doxygen-user, and ~autopsy/docs/doxygen-dev

Comment (markmckinnon) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: From what I see that table never shows up in the docs.  I also checked online in the 4.13 docs and I do not see it as well.

Comment (rcordovano) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: Actually, that makes sense - this is not a public class. I'll accept the PR as is.

**Pull request merged**